src/network/connectionthreads.cpp

```diff
@@ -1157,13 +1157,14 @@ SharedBuffer<u8> ConnectionReceiveThread::handlePacketType_Control(Channel *chan
 				// rtt miscalculation we handle it here
 				float rtt = 0.0f;
 				if (current_time > p.absolute_send_time) {
-					rtt = (current_time - p.absolute_send_time) / 1000.0f;
-				} else if (p.totaltime > 0) {
-					rtt = p.totaltime;
-				}
+					float rtt = (current_time - p.absolute_send_time) / 1000.0;
```

Comment:
`rtt = (current_time - p.absolute_send_time) / 1000.0;`
As rtt is already initialised outside the conditional (L1158).

Reply:
This code is correct.
The variable in the outside scope was not there before 968ce9a, so this PR removed it again.

Reply:
Ok.